manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.installer.yaml

```diff
@@ -0,0 +1,21 @@
+PackageIdentifier: Mumble.Mumble.Server
+PackageVersion: 1.5.634
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-05-19
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
Install in Sandbox via `powershell .\Tools\SandboxTest.ps1 manifests\m\Mumble\Mumble\Server\1.5.634` installs and starts up/runs fine for me.